desktop_version/CMakeLists.txt

```diff
@@ -101,6 +105,17 @@ ELSE()
 	ADD_EXECUTABLE(vvvvvv ${VVV_SRC})
 ENDIF()
 
+# Build options
+IF(ENABLE_WARNINGS)
+	# The weird syntax is due to CMake generator expressions.
+	# Saves quite a few lines and boilerplate at the price of readability.
+	TARGET_COMPILE_OPTIONS(vvvvvv PRIVATE
```

Comment:
Whew, weird is right... but if this works on CentOS 7's GCC/CMake then whatever.

Reply:
That would be worth double checking indeed. I can test on Ubuntu 14.04 if you want :)

Reply:
Ha, yeah that'd be a good test! If it works there I'll merge.

Reply:
Finally managed to test it in Ubuntu 14.04 i386. It only provides SDL2 2.0.2 and CMake 2.8, so I had to hack a bit for proper detection of the libraries, and tweak the code to let it build even though it uses one function from later SDL versions. Seems to build without error.